src/config/dynamic-theme-fixes.config

```diff
@@ -6814,6 +6814,23 @@ img[src$="MCCP-wordmark-denim.png"]
 
 ================================
 
+coupang.com
+mc.coupang.com
+
+INVERT
+img[src="https://static.coupangcdn.com/image/coupang/common/pc_header_img_sprite_new_gnb.svg#person"]
+img[src="https://static.coupangcdn.com/image/coupang/common/pc_header_img_sprite_new_gnb.svg#cart"]
+.main-section h2
+#categoryBestMenu > .category-anchor
+.category-best-title-ad-product
+
+CSS
+.select--category--button {
```

Comment:
I can not approve this part as it will break light scheme using in Dark Reader:
![obraz](https://github.com/user-attachments/assets/e8636421-9bc1-426e-8d58-14ea500012cb)

What I can accept is moving `.select--category--button` to INVERT section.

It will have some black and some white part but at lease it will be light and dark in schemes.

Reply:
Understood.